go/op_exporter/main.go

```diff
@@ -86,13 +98,53 @@ func main() {
 	})
 	go getRollupGasPrices()
 	go getBlockNumber(&health)
+	if *enableK8sQuery {
+		client, err := k8sClient.Newk8sClient()
+		if err != nil {
+			log.Fatal(err)
+		}
+		go getSequencerVersion(&health, client)
+	}
+	//go getAllowedMethods(&health, client)
 	log.Infoln("Listening on", *listenAddress)
 	if err := http.ListenAndServe(*listenAddress, nil); err != nil {
 		log.Fatal(err)
 	}
 
 }
 
+func getSequencerVersion(health *healthCheck, client *kubernetes.Clientset) {
+	ns, err := ioutil.ReadFile("/var/run/secrets/kubernetes.io/serviceaccount/namespace")
+	if err != nil {
+		log.Fatalf("Unable to read namespace file: %s", err)
+	}
+	for {
+		getOpts := metav1.GetOptions{
+			TypeMeta:        metav1.TypeMeta{},
+			ResourceVersion: "",
+		}
+		sequencerStatefulSet, err := client.AppsV1().StatefulSets(string(ns)).Get(context.TODO(), "sequencer", getOpts)
+		if err != nil {
+			panic(err.Error())
```

Comment:
I don't know if we want to panic here, it could cause the application to get into a crashloop backoff and then the health api would be offline. I think it would be best to log an error and then sleep + `continue`

Reply:
Makes sense, updated!